Misc/NEWS.d/next/Library/2022-06-09-14-44-21.gh-issue-93626.amongus.rst

```diff
@@ -0,0 +1 @@
+Set ``__future__.annotations`` to have a ``None`` mandatoryRelease to indicate that it is currently 'TBD'.
```

Comment:
The filename of this news file is `2022-06-09-14-44-21.gh-issue-93626.amongus.rst` which contains a trademark

Reply:
Wow, that's unlucky for a randomly generated string to contain a trademark 😳

Reply:
It's probably less headache-inducing to go ahead and rename it.